packages/live-share/src/internals/RoleVerifier.ts

```diff
@@ -8,7 +8,7 @@ import { waitForResult } from './utils';
 import { RequestCache } from './RequestCache';
 
 const EXPONENTIAL_BACKOFF_SCHEDULE = [100, 200, 200, 400, 600];
-const CACHE_LIFETIME = 60 * 60 * 1000;
+const CACHE_LIFETIME = 5 * 60 * 1000;
```

Comment:
```suggestion
// 5 * 60 * 1000 = 5 minutes
const CACHE_LIFETIME = 5 * 60 * 1000;
```
```suggestion
// 5 * 60 * 1000 = 5 minutes
const CACHE_LIFETIME = 300000;
```

If linter doesn't allow magical numbers, then just adding the comment is fine imo

Reply:
Didn't see a lint error but added a comment